pkg/config/loader.go

```diff
@@ -49,9 +49,12 @@ type Loader struct {
 	// The Kubernetes client that will be used to load Kubernetes objects:
 	client kubernetes.Interface
 
-	// The names of the configuration files, in the order that they should be loaded:
+	// The names of the configuration directories and files, in the order that they should be loaded:
 	files []string
 
+	// The names of loaded configuration files:
+	loadedFiles []string
+
```

Comment:
Can we move this field to the config type?

Reply:
:+1: woo, that will solve the watch problem too, on it